arviz/plots/__init__.py

```diff
@@ -4,7 +4,7 @@
 from .densityplot import plot_density
 from .energyplot import plot_energy
 from .forestplot import plot_forest
-from .kdeplot import plot_kde, _fast_kde, _fast_kde_2d
```

Comment:
I introduced this on purpose #235. Sometimes is useful to have access to `_fast_kde` and `_fast_kde_2d`. Although they should be of no interest to most users so I kept the underscore.

Reply:
Ah I see, my apologies. I'll undo this change

Reply:
Thanks! Don't worry, lot of changes recently and is hard to follow all of them.